scripts/submit_solution.ts

```diff
@@ -0,0 +1,83 @@
+import BN from "bn.js";
+import { factory } from "../src/logging";
+
+const log = factory.getLogger("scripts.submitSolution");
+const BatchExchange = artifacts.require("BatchExchange");
+
+const argv = require("yargs")
+  .option("resultFile", {
```

Comment:
A cool idea for a future improvement is to also accept HTTP URLs here so we can skip the `curl ...` call.

Reply:
But the instances web interface requires a password right?

Reply:
It doesn't require a password. However, I think the primary use case would be to upload a local file since files that are put in the S3 bucket have likely already been submitted by the driver.